scripts/Dockerfile.devel

```diff
@@ -1,12 +1,19 @@
 FROM ceph/ceph:v15
 
+RUN \
+ mkdir -p /usr/local/go && \
+ curl https://storage.googleapis.com/golang/go1.13.9.linux-amd64.tar.gz | tar xzf - -C /usr/local/go --strip-components=1
+
 ENV GOPATH=/go \
-	GO111MODULE=on
+ GOROOT=/usr/local/go \
+ GO111MODULE=on
+
+ENV PATH=$GOROOT/bin:$GOPATH/bin;$PATH
```

Comment:
can this ENV be added to the above ENV layer?

Reply:
Done.